Mathlib/Topology/Instances/CantorSet.lean

```diff
@@ -313,3 +317,41 @@ theorem cantorSet_eq_zero_two_ofDigits :
     exact ofDigits_zero_two_sequence_mem_cantorSet ha.left
 
 end ternary02
+
+/-!
+## The Cantor set is homeomorphic to `ℕ → Bool`
+-/
+
+open Real
+
+/-- Canonical bijection between the Cantor set and infinite binary tree. -/
+noncomputable def cantorSet_equiv_nat_to_bool : cantorSet ≃ (ℕ → Bool) where
```

Comment:
this is what I meant about lowerCamelCase (since it's a `def`)
```suggestion
/-- Canonical bijection between the Cantor set and infinite binary tree. -/
noncomputable def cantorSetEquivNatToBool : cantorSet ≃ (ℕ → Bool) where
```